src/WMSTileLayer.js

```diff
@@ -32,9 +32,8 @@ class WMSTileLayer extends GridLayer<LeafletElement, Props> {
   }
 
   getOptions(params: Object): Object {
-    return reduce(
-      super.getOptions(params),
-      (options, value, key) => {
+    return Object.entries(super.getOptions(params)).reduce(
```

Comment:
Could you use `Object.keys()` here please? I think there are more browsers supporting it than `Object.entries()`.